content/docs/releases/release-notes/release-notes-1.16.md

```diff
@@ -63,6 +63,27 @@ Users who use IAM Roles for Service accounts or Pod Identity need
 not specify the region, but if your Issuer or ClusterIssuer does include a region (for the sake of satisfying the old API validation),
 that issuer region will be ignored, if the `AWS_REGION` environment variable is set.
 
+cert-manager will now use regional STS endpoints, when using `AssumeRole` or when
+using a dedicated (non-mounted) Kubernetes ServiceAccount.
+The regional endpoint will be computed based on the Issuer `region` field,
+or the `AWS_REGION` environment variable.
+
+> ℹ️ This change only affects the `AssumeRole` configuration, which is used for cross-account authentication,
+> and the `AssumeRoleWithWebIdentity` configuration, where the user supplies the name of a Kubernetes ServiceAccount.
+> If does not affect you if you have configured the cert-manager ServiceAccount for IRSA,
+> where the ServiceAccount token is mounted in to the cert-manager controller Pod.
+> Regional STS endpoints were already being used in that case.
+>
+> ℹ️ There are good reasons to use regional STS endpoints, summarized as follows on the [Amazon AWS blog](https://aws.amazon.com/blogs/security/how-to-use-regional-aws-sts-endpoints/):
+>
+> > Although the global (legacy) AWS STS endpoint https://sts.amazonaws.com is highly available, it’s hosted in a single AWS Region—US East (N. Virginia)—and like other endpoints, it doesn’t provide automatic fail-over to endpoints in other Regions.
```

Comment:
suggestion (non-blocking): I realise this is a quote, but I found the section in hyphens difficult to read without spaces. Might just be personal preference though!

```suggestion
> > Although the global (legacy) AWS STS endpoint https://sts.amazonaws.com is highly available, it’s hosted in a single AWS Region — US East (N. Virginia) — and like other endpoints, it doesn’t provide automatic fail-over to endpoints in other Regions.
```